mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -62,6 +63,28 @@
 event_id = {'Sync On': 5}
 
 
+def test_orig_units():
+    """Test exposure of original channel units."""
+    with pytest.warns(RuntimeWarning, match='will be dropped'):
+        raw = read_raw_brainvision(vhdr_path)
+    orig_units = raw._orig_units
+    assert len(orig_units) == 32
+    assert orig_units['FP1'] == u'µV'
+    assert orig_units['CP5'] == 'BS'
+    assert orig_units['CP6'] == u'µS'
+    assert orig_units['HL'] == 'ARU'
+    assert orig_units['HR'] == 'uS'
+    assert orig_units['Vb'] == 'S'
+    assert orig_units['ReRef'] == 'C'
+
+    # Also test handling in base.py
+    # Should complain that channel Cz does not have a corresponding original
```

Comment:
channel not_Cz

Reply:
no this is actually true ... I initialize a raw object with a channel Cz ... but the corresponding channel in `orig_units` is called not_Cz, so we get a complaint, that for channel Cz, no corresponding original unit was found.